pkg/lib/config/feature.go

```diff
@@ -26,7 +26,8 @@ var _ = FeatureConfigSchema.Add("FeatureConfig", `
 		"messaging": { "$ref": "#/$defs/MessagingFeatureConfig" },
 		"collaborator": { "$ref": "#/$defs/CollaboratorFeatureConfig" },
 		"web3": { "$ref": "#/$defs/Web3FeatureConfig" },
-		"admin_api": { "$ref": "#/$defs/AdminAPIFeatureConfig" }
+		"admin_api": { "$ref": "#/$defs/AdminAPIFeatureConfig" },
+		"feature": { "$ref": "#/$defs/FixtureFeatureConfig" }
```

Comment:
```yaml
test_mode:
  fixed_oob_otp:
    enabled: true
    code: "123456"
```

A second thought, I think we cannot have fixed link_otp. It is because we can look up a particular reset password flow by a link_otp. If every link_otp is the same, then reset password of different user will overwrite each other. Seems we need to agree on a deterministic 32-character length code that the client can generate by themselves.

And recovery code is actually not fixed. But we want to allow a magic recovery code that would verified successfully.